Sources/WebDriver/Requests.swift

```diff
@@ -663,12 +663,19 @@ public enum Requests {
     }
 
     // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#status
-    public struct Status: Request {
+    public struct Status_Legacy: Request {
```

Comment:
How about using an `enum` for the namespacing?

```swift
enum Legacy {
  public struct Status: Request { ... }
}

enum W3C {
  public struct Status: Request { ... }
}
```

Reply:
Good idea. Done in latest